pwndbg/next.py

```diff
@@ -88,6 +89,16 @@ def break_next_call(address=None):
         if capstone.CS_GRP_CALL in ins.groups:
             return ins
 
+def break_call_regex(regex=''):
+    while pwndbg.proc.alive:
+        ins = break_next_call()
+
+        if not ins:
+            break
+
+        if re.search(regex, ins.symbol):
```

Comment:
I think `re.match` would be better, so that if I want to stop on `printf`, it doesn't accidentally stop on e.g. `sprintf`